.github/workflows/github.yml

```diff
@@ -145,6 +145,14 @@ jobs:
       with:
         submodules: recursive
 
+    - name: Ensure LF line endings
+      if: ${{ startsWith(matrix.preset, 'linux-clang-test') }}
+      run: |
+        find . -path ./.git -prune -o -path ./AI/FuzzyLite -prune -o -path ./android -prune \
+        -o -path ./osx -prune -o -path ./test/googletest -prune -o -type f \
+        -not -name '*.png' -and -not -name '*.vcxproj*' -and -not -name '*.props' \
+        -not -name '*.wav' -not -name '*.ico' -print0 | ( ! xargs -0 grep -l -z -P '\r\n' )
```

Comment:
Parenthesis create a subshell. I do not think you need to spawn a new subshell and grouping with curly braces is sufficient to achieve the effect you want, no?